core/src/test/java/org/elasticsearch/search/aggregations/InternalAggregationTestCase.java

```diff
@@ -122,4 +124,16 @@ protected final T createUnmappedInstance(String name) {
     protected NamedWriteableRegistry getNamedWriteableRegistry() {
         return namedWriteableRegistry;
     }
+
+    /**
+     * @return a random {@link DocValueFormat} that can be used in aggregations which
+     * compute numbers.
+     */
+    protected static DocValueFormat randomNumericDocValueFormat() {
+        final List<Supplier<DocValueFormat>> formats = new ArrayList<>(3);
+        formats.add(() -> DocValueFormat.RAW);
+        formats.add(() -> DocValueFormat.BOOLEAN);
```

Comment:
I kept this one because it was used in many aggregations tests before and seems to be supported (true/false are translated to 1/0) by aggregations that work on numeric. I'm not sure it makes much sense though. Would you like to see it removed?